src/shared/components/Select/Select.style.ts

```diff
@@ -23,13 +23,13 @@ export const SelectButton = styled.button<SelectButtonProps>`
   width: 100%;
   overflow: hidden;
   text-overflow: ellipsis;
-  border: none;
   background: none;
   color: ${({ filled }) => (filled ? colors.gray[50] : colors.gray[300])};
   text-align: left;
   display: flex;
   justify-content: space-between;
   align-items: center;
+  border: ${({ isOpen }) => (isOpen ? `1px solid ${colors.blue[500]} !important` : 0)};
```

Comment:
Why is `!important` needed here?

Reply:
Because without this it is overwritten by FormGroup styles
![Zrzut ekranu 2021-09-3 o 11 03 56](https://user-images.githubusercontent.com/13706246/131980198-394def49-c0ec-46db-98d2-658afa1d30a4.png)

Reply:
Oh, that's a bummer. I think if we tweaked `FormGroup` styles we could resolve this. Could you try removing that `:not(:focus)` selector in `InputBase.styles.ts:46` and just apply the border directly to `input, button, textarea`? I think that should reduce the specificity and make this work without `important`

Reply:
Nah, still the same
<img width="518" alt="Zrzut ekranu 2021-09-3 o 11 28 38" src="https://user-images.githubusercontent.com/13706246/131983716-25814fb5-7421-4b70-a31e-94baff4b06ad.png">

Maybe we could remove button selector from `FormGroup` styles and apply styles only to `Select`? Select is the only possible button in FormGroup and already has bunch of own styles.

Reply:
Okay, let's try that. But only for `border` styles